### ISSUE TITLE
[Bug]: Unable to change button focus outline color using react-tabster

### ISSUE THREAD

**Issue by chieftn** [microsoft/fluentui#27943]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
Outlined here: https://codesandbox.io/s/laughing-greider-6476ue?file=/example.tsx:176-309
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/s/laughing-greider-6476ue?file=/example.tsx:176-309

### Bug Description

I am attempting to change button focus outline color but cannot successfully complete.  I have provided the following code sandbox to assist with repro: https://codesandbox.io/s/laughing-greider-6476ue?file=/example.tsx:176-309.  I've tried two variations using focus and focus-within.  The steps I have taken appear to be in line with documentation but have no effect on the focus indicator.  Looking for guidance / advisement.  Thank you.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by layershifter** [comment]
@chieftn `Button` uses a different helper to define styles (`createCustomFocusIndicatorStyle`):

https://github.com/microsoft/fluentui/blob/ab53a4c86a49a4d4268f3d87809bee263f818c5d/packages/react-components/react-button/src/components/Button/useButtonStyles.styles.ts#L111-L120

The simplest override for a color would be:

```ts
const useStyles = makeStyles({
  focusIndicator: createCustomFocusIndicatorStyle({
    outlineColor: "red"
  })
});
```

https://codesandbox.io/s/awesome-bash-yz54tw?file=/example.tsx:152-267

**Comment by chieftn** [comment]
Acknowledged -- thank you very much for your assistance.
